grobid-core/src/test/java/org/grobid/core/engines/FullTextParserTest.java

```diff
@@ -1,30 +1,42 @@
 package org.grobid.core.engines;
 
+import org.apache.commons.lang3.tuple.Pair;
 import org.grobid.core.analyzers.GrobidAnalyzer;
```

Comment:
There are quite a few unused imports. Might be good to tidy it up (and remove commented out code, perhaps mark test to be ignored instead).